cluster/examples/kubernetes/cassandra/cluster.yaml

```diff
@@ -70,6 +70,9 @@ metadata:
   name: rook-cassandra
   namespace: rook-cassandra
 spec:
+  # A key/value list of annotations
```

Comment:
could we put optional crd settings at the end of the yaml? It's distracting to readability of the critical settings.